packages/expo-video/ios/VideoPlayer.swift

```diff
@@ -86,7 +86,10 @@ internal final class VideoPlayer: SharedRef<AVPlayer>, Hashable, VideoPlayerObse
     observer.cleanup()
     NowPlayingManager.shared.unregisterPlayer(self)
     VideoManager.shared.unregister(videoPlayer: self)
-    pointer.replaceCurrentItem(with: nil)
+
+    DispatchQueue.main.async { [pointer] in
```

Comment:
Add a comment why it's necessary to switch to the main thread, i.e. describe the crash and maybe how to reproduce it